content/docs/user-guide/assets/asset-processor/debugging.md

```diff
@@ -6,9 +6,256 @@ weight: 500
+
+If other content or logic requires the changes from processing the asset, other team members may run into problems using the Product Assets output by the older version of the builder with the other updated logic.
+
+#### Immediate Solution
+Change the version number of the builder reported in that builder's AssetBuilderSDK::AssetBuilderDesc.
+
+#### Long Term Solutions
+Create automated jobs that process all assets cleanly at some interval, to catch issues like this.
+
+One configuration that would catch issues would be:
+* Iterative Job - Processes assets after a code rebuild.
+* Clean Job - Cleans and processes all assets after a code build.
+* Compare the hashes of each Product Asset between builds and runs of these jobs. If any assets end up different in two back to back Clean Jobs but did not process in the Iterative Job, this usually means an asset builder was modified without changing its version.
+
+#### Debugging
+Debugging this issue is usually focused around discovery of the issue in the first place. Once it's identified as a problem, identifying which builder needs a version change and making that change is usually trivial.
+
+If some team members are experiencing issues with content, but not everyone on the team is having the problem, but that content has not changed recently, this is commonly the cause. Check if the builder for that content has been modified in source control recently, and if so, check if the version was changed. If not, then it's likely that the builder change without a version number change is causing a problem.
+
+The Asset Processor UI lets you right click a Source Asset in the asset tab and re-run all jobs on that asset. If doing so causes a change in Product Asset, it likely means a builder was changed without having the version updated. If you want to debug this back and force, it's recommended that you back up the old Product Asset(s) before reprocessing the Source Asset.
+
+### Logic in Asset Builder persists between building assets
+
+#### Situation
+In some cases, due to information persisting during asset processing, you may get different Product Asset output from jobs based on the order Source Assets are processed. This is usually due to a bug within the Asset Builder that processed this content.
+
+The Asset Processor creates one or more Asset Builder executable based on configuration settings, and uses these to process assets, sending jobs to these Asset Builder executables to run with the individual Asset Builders. There is no system within the Asset Builder Executable to completely shut down and clear all Asset Builders, so it is possible for an Asset Builder to persist information across multiple sessions of processing jobs for a single Asset Builder Executable.
+
+#### Effect
+Stale data or logic running in an Asset Builder in an Asset Builder Executable can result in an Asset processing incorrectly.

```

Comment:
This reads like a typo but I understand it... Not sure if there's some way to reword it so it doesn't look like you've accidentally doubled "in an Asset Builder".  Maybe just drop the 2nd part, it should be understood that Asset Builders run in AssetBuilder.exe by this point?